src/utils/normalizeScreenshot.js

```diff
@@ -58,18 +62,18 @@ async function normalizeIOSScreenshot(browser, screenDimensions, base64Screensho
   return base64Screenshot;
 }
 
-
-export default async function normalizeSreenshot(browser, screenDimensions, base64Screenshot) {
+export default async function normalizeScreenshot(browser, screenDimensions, base64Screenshot) {
   let normalizedScreenshot = base64Screenshot;
 
   // check if we could have a retina image
   if (screenDimensions.getPixelRatio() > 1) {
     normalizedScreenshot = await normalizeRetinaScreenshot(browser, screenDimensions, normalizedScreenshot);
   }
 
-  // check if we have to crop navigation- & toolbar for iOS
-  if (browser.isMobile && browser.isIOS) {
-    normalizedScreenshot = await normalizeIOSScreenshot(browser, screenDimensions, normalizedScreenshot);
+  // check if we have to crop navigation- & toolbar for iOS or Android
+  if (browser.isMobile) {
```

Comment:
Hmmm, does the `viewportHeight` and `screenHeight` return different values for real device chrome? If not then it should just pass through without cropping. If so then maybe there is a way to check for emulators.